libpod/container_api.go

```diff
@@ -705,3 +705,19 @@ func (c *Container) Sync() error {
 
 	return nil
 }
+
+// Restart restarts a running container and takes a given timeout in uint
+func (c *Container) Restart(timeout uint) error {
+	ctx := context.TODO()
+	state, err := c.State()
```

Comment:
Don't use this - lock the container and do a syncContainer()

Reply:
fixed